lib/auth/Vault.js

```diff
@@ -24,6 +24,10 @@ function vaultSignatureCb(err, authInfo, log, callback, streamingV4Params) {
     const info = authInfo.message.body;
     const userInfo = new AuthInfo(info.userInfo);
     const authorizationResults = info.authorizationResults;
+    const auditLog = { accountDisplayName: userInfo.getAccountDisplayName() };
+    const iamDisplayName = userInfo.getIAMdisplayName();
+    if (iamDisplayName) auditLog.IAMdisplayName = iamDisplayName;
```

Comment:
personally prefer curly braces around if statements (at least when it comes to JS), could also make it more consistent as the others seem to have in the same repo